python/feathub/processors/flink/table_builder/file_system_utils.py

```diff
@@ -67,6 +67,7 @@ def get_table_from_file_source(
         # Set ignore-parse-errors to set null in case of csv parse error
         descriptor_builder.option("csv.ignore-parse-errors", "true")
 
+    t_env.drop_temporary_table(file_source.name)
```

Comment:
Why do we need to drop the table here?

Reply:
StreamTableEnvironment throws an exception when creating temporary tables with the same name so the temporary table is dropped before we create a new one.

After some thought, I think it is better to append a randomly generated part to the temporary table name to avoid name conflict, similar to what we did for Sink FeatureTable. Similar to sink, we will use an anonymous table after the bug in VVR-6.0.2 is fixed so that we can get rid of the `create_temporary_table` altogether.